### PR TITLE
Docs: Stream-line Intro

### PR DESCRIPTION
**Pull request by ax3l**
Stream-line the intro pages and the install start a little more.

### CODE REVIEW

File: docs/source/index.rst [hunk 1]
```diff
@@ -10,24 +10,24 @@
...
 PIConGPU is a fully relativistic, many GPGPU, 3D3V particle-in-cell (PIC) code.
-The Particle-in-Cell algorithm is a central tool in plasma physics.
+The PIC algorithm is a central tool in plasma physics.
 It describes the dynamics of a plasma by computing the motion of electrons and ions in the plasma based on Maxwell's equations.
 
 How to Read This Document
 -------------------------
 
-Generally, you want to follow those pages in-order to get started.
+Generally, **follow the manual's pages in-order** to get started.
```
Comment (PrometheusPi) [inline docs/source/index.rst]: OPTIONAL:
more often used is just: "manual pages"

File: docs/source/index.rst [hunk 2]
```diff
@@ -10,24 +10,24 @@
...
+
+   The online version of this document is **versioned** and shows by default the manual of the last *stable* version of PIConGPU.
+   If you are looking for the latest *development* version, `click here <https://picongpu.readthedocs.io/en/latest/>`_.
+
 
-.. note::
-   We also have a `wiki`_
-   and a general `official homepage`_
+We are migrating our `wiki`_ to this manual, but some pages might still be missing.
+We also have a general `official homepage`_ .
```
Comment (PrometheusPi) [inline docs/source/index.rst]: OPTIONAL:
just "an official homepage"

File: docs/source/install/path.rst [hunk 3]
```diff
@@ -11,17 +11,22 @@ The first part is usually the job of a system administrator while the second par
...
+Ways to Install
+---------------
+
+Choose *one* of the install methods below to get started:
+
+Load Modules
+^^^^^^^^^^^^
+
+On HPC systems and clusters, software is usually provided by system administrators via a module system (e.g. [modules]_, [Lmod]_).
+In case our :ref:`software dependencies <install-dependencies>` are available, we usually create a file in our ``$HOME`` named :ref:`<queueName>_picongpu.profile <install-profile>` and load according modules and set :ref:`helper environment variables <install-dependencies-picongpu>` within.
```
Comment (PrometheusPi) [inline docs/source/install/path.rst]: OPTIONAL:
double "and"
better:
```
...create a file in our ``$HOME`` named :ref:`<queueName>_picongpu.profile <install-profile>`, load according modules, and set ...
```

Comment (ax3l) [reply docs/source/install/path.rst]: Sure! I will split the sentence, it's too long anyway.

**Comment by ax3l** [comment]
@PrometheusPi updated, thx for the comments!

**Comment by PrometheusPi** [comment]
There is an issue with the compile suite.

**Pull request merged**